.github/workflows/release.yml

```diff
@@ -0,0 +1,55 @@
+name: Release
+on:
+  push:
+    tags:
+      - "v*"
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set release_tag
+      run: echo "RELEASE_TAG=${GITHUB_REF:10}" >> $GITHUB_ENV
+    - name: Install kustomize
+      run: make kustomize
+    - name: Install envsubst
+      run: make envsubst
+    - name: Docker login
+      run: docker login -u $DOCKER_USER -p $DOCKER_PASSWORD $DOCKER_REGISTRY
+      env:
+        DOCKER_USER: ${{secrets.DOCKER_USER}}
+        DOCKER_PASSWORD: ${{secrets.DOCKER_PASSWORD}}
+        DOCKER_REGISTRY: ${{ secrets.DOCKER_REGISTRY }}
+    - name: build and push docker image
+      run: |
+        make docker-build
+        make docker-push
+        docker image prune -a -f
+      env:
+        IMG: ${{ secrets.DOCKER_REGISTRY_REPO }}/cluster-api-outscale-controllers:${{ env.RELEASE_TAG }}
+        RELEASE_TAG: ${{ env.RELEASE_TAG }}
+        DOCKER_BUILDKIT: 1
+    - name: generate artefacts
+      run: make release
+      env:
+        IMG_RELEASE: ${{ secrets.DOCKER_REGISTRY_REPO }}/cluster-api-outscale-controllers:${{ env.RELEASE_TAG }}
+        RELEASE_TAG: ${{ env.RELEASE_TAG }}
+    - name: gh login
+      run: make gh-login
+      env:
+        SECRET_GITHUB_TOKEN: ${{ secrets.SECRET_GITHUB_TOKEN }}
+    - name: generate changelog
+      run: make release-changelog
+      env:
+        IMG_RELEASE: ${{ secrets.DOCKER_REGISTRY_REPO }}/cluster-api-outscale-controllers:${{ env.RELEASE_TAG }}
+        RELEASE_TAG: ${{ env.RELEASE_TAG }}
+        PREVIOUS_RELEASE_TAG: ${{ secrets.PREVIOUS_RELEASE_TAG }}
```

Comment:
You would have to update the secret every release, could we not compute this ?

Reply:
Done